.github/workflows/version_updated.yml

```diff
@@ -3,6 +3,12 @@ on:
     types: [opened, reopened, synchronize]
     branches:
       - "main"
+    paths:
```

Comment:
Nice! Would we want to include Dockerfile, priv, assets, rel, run.sh here?

Reply:
good call will update

Reply:
not only this one but the others 👍

Reply:
kept the test workflow focused on exs